### DISCUSSION TITLE
When I made the build with video.js, did my project get too heavy, normal?

### DISCUSSION THREAD

**Discussion by vasconx** [vercel/next.js#15951] [Help]
Hi, I'm developing a pwa site with next.js, I think that only someone with more experience with web development and next.js will be able to help me.
I'm using the video.js package to make a personalized player available on the site. However, when executing the build of next.js, I saw the package changed too much the file size of the page it is on. And I don't know if this is normal. If it is normal in web development, these players behave like this and if that can be a problem. Or if video.js happens to have some unnecessary dependency. Because it gave me an alert to use or not to use the package because I don't want a slow website because it uses files intensively. I'll show you the photos from before and after using the video.js package in my project and you tell me what you think.
![BEFORE video.js](https://user-images.githubusercontent.com/48503371/89566925-3e47ac80-d7f7-11ea-85a7-2229d73beadb.png)
![AFTER video.js](https://user-images.githubusercontent.com/48503371/89566928-40aa0680-d7f7-11ea-8325-fe96644a9113.png)

**Comment by vasconx** [comment]
oh boy

**Comment by timneutkens** [comment]
Looking at bundlephobia video.js is super heavy yeah: https://bundlephobia.com/result?p=video.js@7.8.4

**Reply by karthik-js** [reply]
@timneutkens can you suggest any alternatives which are lighter and can be used with next js?
